molecule/default/converge.yml

```diff
@@ -2,6 +2,9 @@
 - hosts: all
   become: true
 
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
```

Comment:
Is this necessary?
I hope not, in which case I would prefer to drop it

Reply:
Yes it is for the molecule tests where this is not set. I copied this from one of the other role packages